src/algebra/algebra/spectrum.lean

```diff
@@ -212,6 +213,26 @@ theorem preimage_units_mul_eq_swap_mul {a b : A} :
   (coe : Rˣ → R) ⁻¹' σ (a * b) = coe ⁻¹'  σ (b * a) :=
 by { ext, exact unit_mem_mul_iff_mem_swap_mul, }
 
+section star
+
+variables [star_add_monoid R] [star_ring A] [star_module R A]
+
+local postfix `⋆`:std.prec.max_plus := star
+
+lemma star_mem_resolvent_set {r : R} {a : A} (h : r ∈ resolvent_set R a) :
```

Comment:
```suggestion
lemma star_mem_resolvent_set_star {r : R} {a : A} (h : r ∈ resolvent_set R a) :
```

Reply:
Also, I'm not sure this should be in the `spectrum` namespace, given `resolvent_set` isn't

Reply:
name changed. As for namespace, everything else regarding the resolvent set is in the spectrum namespace currently. So, if we wanted to move it, we could, but we should move everything in that case.